src/main/groovy/org/broadinstitute/orsp/ConsentGroupExtraProperties.groovy

```diff
@@ -0,0 +1,81 @@
+package org.broadinstitute.orsp
+
+class ConsentGroupExtraProperties {
+    // General Data
+    String startDate = ""
+    String endDate = ""
+    String onGoingProcess = ""
+    String source = ""
+    String collInst = ""
+    String collContact = ""
+    String consent = ""
+    String protocol = ""
+    String institutionalSources = ""
+    String describeConsentGroup = ""
+    String requireMta = ""
+    String projectReviewApproved = "false"
+
+    // Info Security
+    String pii = ""
+    String compliance = ""
+    String textCompliance = ""
+    String sensitive = ""
+    String textSensitive = ""
+    String accessible = ""
+    String textAccessible = ""
+
+    // Sharing data
+    String sharingPlan = ""
+    String databaseControlled = ""
+    String databaseOpen = ""
+
+    // InternationalCohorts
+    String individualDataSourced = ""
+    String isLinkMaintained = ""
+    String feeForServiceWork = ""
+    String areSamplesComingFromEEAA = ""
+    String isCollaboratorProvidingGoodService = ""
+    String isConsentUnambiguous = ""
+
+    transient ConsentGroupExtraProperties buildConsentGroupExtraProperties(Map<String, List<String>> extraPropsMap) {
+        if(!extraPropsMap.isEmpty()) {
+            // General Data
+            this.setStartDate(extraPropsMap.get(IssueExtraProperty.START_DATE, [""]).first())
```

Comment:
My concern here is that there is a ton of duplicated code (both here and in the `ProjectExtraProperties` class). We already do this kind of thing in [`Issue`](https://github.com/broadinstitute/orsp-pub/blob/bb6507f310a87f1ba4875f6fbee001bca26cfb5c/grails-app/domain/org/broadinstitute/orsp/Issue.groovy#L69) and, to some degree, it makes more sense for all of this code to be using that mechanism instead of replicating this logic here.

Reply:
I think it would be simpler to make the constructor take an `Issue` object and then populate the values from what is already known on the issue itself.